pkg/agent/run.go

```diff
@@ -176,27 +178,35 @@ func gatherAndPostData(ctx context.Context) {
 	log.Println("Running Agent...")
 	log.Println("Posting data to ", baseURL)
 	if config.OrganizationID == "" {
```

Comment:
I think we might need to handle the case when the org is set too?

Reply:
Yes, I think this should be a part of the larger changes made with the load-from-file addition